backend/src/v2/driver/main.go

```diff
@@ -27,9 +27,9 @@ import (
 	"github.com/golang/glog"
 
 	"github.com/golang/protobuf/jsonpb"
-	pb "github.com/kubeflow/pipelines/api/v2alpha1/go"
 	"github.com/kubeflow/pipelines/backend/src/v2/common"
 	"github.com/kubeflow/pipelines/backend/src/v2/common/mlmd"
+	pb "github.com/kubeflow/pipelines/backend/src/v2/third_party/pipeline_spec"
```

Comment:
`backend/src/v2` uses out-dated pipelinespec. The fix of v2 package takes time. As a workaround, I temporarily pasted out-dated pipeline spec that v2 is using.

Reply:
Thanks for the catch! I will update pipeline spec in a separate PR.